packages/instant/src/util/coinbase_api.ts

```diff
@@ -0,0 +1,10 @@
+import { BigNumber } from '@0xproject/utils';
+
+const baseEndpoint = 'https://api.coinbase.com/v2';
+export const coinbaseApi = {
```

Comment:
is coinbase the best choice for this?

Reply:
I'm not sure, jw what thought process you had when choosing the coinbase API to supply the price. Some things to consider, support for other currencies for localization, rate limiting, downtime, etc.

Reply:
I looked into a few options but did not do a crazy thorough comparison. Coinbase has 3 req / sec, 10k req / hour per IP, which seems reasonable. They also support other (fiat) currencies. Not much info available on downtime. Do you know of other APIs?